restx-core/src/main/java/restx/http/ExpiresHeaderFilter.java

```diff
@@ -41,7 +41,7 @@ protected void onEntityOutput(StdRoute stdRoute, RestxRequest req, RestxResponse
 
         ExpiresAfter expiresAfterAnn = operationDescription.findAnnotation(ExpiresAfter.class).get();
 
-        Locale currentLocale = currentLocaleResolver.guessLocale(req);
+        Locale currentLocale = Locale.US;
```

Comment:
@nroulon I guess you can then remove the `currentLocaleResolver` injected component into `ExpiresHeaderFilter`

Reply:
Indeed, I remove `currentLocaleResolver`